src/openzaak/components/catalogi/tests/admin/test_published_readonly.py

```diff
@@ -80,13 +80,13 @@ def test_readonly_zaaktype(self, m):
 
         # check custom formatting
         procestype = response.html.find(class_="field-selectielijst_procestype").div.div
-        self.assertEqual(procestype.text, "1 - Instellen en inrichten organisatie")
+        self.assertTrue("1 - Instellen en inrichten organisatie" in procestype.text)
         behandeling = response.html.find(
             class_="field-doorlooptijd_behandeling"
         ).div.div
-        self.assertEqual(
-            behandeling.text,
-            ngettext_lazy("{days} day", "{days} days", 10).format(days=10),
+        self.assertTrue(
+            ngettext_lazy("{days} day", "{days} days", 10).format(days=10)
```

Comment:
Why now in instead of equals?

Reply:
In django 4.2 they added additional `div` with `class="flex-container"` inside the field template, so now we need either to add one extra `.div` when finding the html element or change the assetion. I used the latter approach. Tbh I don't remember why the tests for formatting were added, but it looks like we test a django template here. So I think we can remove them in the future

Reply:
Got it